GenderPayGap.UnitTests/GenderPayGap.WebUI.Tests/Builders/OrganisationScopeBuilder.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using GenderPayGap.Core;
+using GenderPayGap.Core.Classes;
+using GenderPayGap.Database;
+
+namespace GenderPayGap.WebUI.Tests.Builders
+{
+    public class OrganisationScopeBuilder
+    {
+
+        private Organisation organisation = new OrganisationBuilder();

```

Comment:
It took me a while to work out how this could possibly compile!
While the `implicit operator` stuff is very **clever**/**cool**, I don't think it's very **clear**.
Unless there's a good reason, I'd rather we always used `thingBuilder.Build()`

Reply:
Sure, no worries! I agree that it's clearer - I just looked at a few guides on the builder pattern specific to C# and a handful recommended the implicit operator, so I went with it! Normally I'd always use `.Build()` but I wasn't sure if this was a best practice in C# since a lot of places recommended it!

Reply:
Oh that's interesting! Well done for doing the research 😄 

The implicit operator is very clever and does make the code more succinct, which can be nice.
But it can be surprising, and I favour clarify much more than succinctness.

I can why it would be worth it for a lot of organisations - e.g. if we were using loads of builders, and if the organisation did mainly C# so implicit operators were common in the code and wouldn't be surprising.

Reply:
Yeah, makes sense. I'm also in favour of `.Build()` for clarity over cool-ness, but I can see how this stuff could be second-nature to C# programmers and end up being recommended for the builder pattern if they're already familiar with it!